### PR TITLE
Fix 21891: Add default-context-path test in an ear application with a war

### PR DESCRIPTION
**Pull request by shingwaichan**

### CODE REVIEW

**Review by edburns — DISMISSED** [review]
Make sure copyright date is correct.  2017 only for new tests.

Make sure to update the copyright-excludes for the new build.properties

File: appserver/tests/appserv-tests/devtests/web/servlet-4.0/defaultContextPathEar/WebTest.java [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+ * DO NOT ALTER OR REMOVE COPYRIGHT NOTICES OR THIS HEADER.
+ *
+ * Copyright (c) 1997-2017 Oracle and/or its affiliates. All rights reserved.
+ *
+ * The contents of this file are subject to the terms of either the GNU
+ * General Public License Version 2 only ("GPL") or the Common Development
+ * and Distribution License("CDDL") (collectively, the "License").  You
+ * may not use this file except in compliance with the License.  You can
+ * obtain a copy of the License at
+ * https://glassfish.dev.java.net/public/CDDL+GPL_1_1.html
```
Comment (arindam-bandyopadhyay) [inline appserver/tests/appserv-tests/devtests/web/servlet-4.0/defaultContextPathEar/WebTest.java]: Please use newer version of copyright header. e.g.
https://github.com/javaee/glassfish/blob/master/appserver/tests/appserv-tests/devtests/web/servlet-4.0/compositeTrailerSupplier/WebTest.java

File: appserver/tests/appserv-tests/devtests/web/servlet-4.0/defaultContextPathEar/build.xml [hunk 2]
```diff
@@ -0,0 +1,121 @@
...
+    DO NOT ALTER OR REMOVE COPYRIGHT NOTICES OR THIS HEADER.
+
+    Copyright (c) 1997-2017 Oracle and/or its affiliates. All rights reserved.
+
+    The contents of this file are subject to the terms of either the GNU
+    General Public License Version 2 only ("GPL") or the Common Development
+    and Distribution License("CDDL") (collectively, the "License").  You
+    may not use this file except in compliance with the License.  You can
+    obtain a copy of the License at
+    https://glassfish.dev.java.net/public/CDDL+GPL_1_1.html
```
Comment (arindam-bandyopadhyay) [inline appserver/tests/appserv-tests/devtests/web/servlet-4.0/defaultContextPathEar/build.xml]: Please use newer version of copyright header. e.g.
https://github.com/javaee/glassfish/blob/master/appserver/tests/appserv-tests/devtests/web/servlet-4.0/compositeTrailerSupplier/build.xml
This comment is applicable for all java and xml files

**Review by edburns — APPROVED** [review]

**Comment by shingwaichan** [comment]
@glassfishrobot Run CI tests please

**Comment by shingwaichan** [comment]
@glassfishrobot Run CI tests please

**Comment by shingwaichan** [comment]
@glassfishrobot Run CI tests please

**Pull request merged**